From 5ed3b53a8f3f12085c96e5015fd8bd301fdca920 Mon Sep 17 00:00:00 2001
From: Zhou Peng <eagle.zhou@nxp.com>
Date: Tue, 10 Jul 2018 18:39:08 +0800
Subject: [PATCH 4085/5242] MLK-18841 - [i.MX8MQ/Hantro]: update default major
 number

commit  a57859f811f355b7ba297162eeb635b81cbd9e73 from
https://source.codeaurora.org/external/imx/linux-imx.git

In the latest kernel, default major number '-1' will lead to probe failure
Need to set default value '0' for dynamic major.

Signed-off-by: Zhou Peng <eagle.zhou@nxp.com>
(cherry picked from commit a58b2107b61436414b965740a1c7167b619e1b17)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/hantro/hantrodec.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/hantro/hantrodec.c b/drivers/mxc/hantro/hantrodec.c
index c1406b5..0346a06 100755
--- a/drivers/mxc/hantro/hantrodec.c
+++ b/drivers/mxc/hantro/hantrodec.c
@@ -159,7 +159,7 @@
 	} while (0)
 
 
-static int hantrodec_major = -1; /* dynamic allocation */
+static int hantrodec_major; /* dynamic allocation */
 
 /* here's all the must remember stuff */
 typedef struct {
-- 
1.7.9.5

